class.jetpack-gutenberg.php

```diff
@@ -151,18 +178,39 @@ public static function register( $slug, $args, $availability ) {
 		}
 	}
 
+	/**
+	 * Set a (non-block) extension as available
+	 *
+	 * @param string $slug Slug of the extension.
+	 */
+	public static function set_extension_available( $slug ) {
+		self::$availability[ self::remove_extension_prefix( $slug ) ] = true;
```

Comment:
I was surprised that we are removing the prefix instead of enforcing it here. did you make a decision to do this or is this necessary to remain compatible with existing systems somehow? would there be benefit to requiring the full string here?

Reply:
> I was surprised that we are removing the prefix instead of enforcing it here. did you make a decision to do this or is this necessary to remain compatible with existing systems somehow?

The latter, and I don't like it. It's mostly since the entries in `index.json` are still unprefixed, as is the endpoint's response. I was considering changing that as part of this PR as well, but it seemed big enough already (even though it's exposing more and more how annoying all the adding and removing of the prefixes is).

Anyway, right now, this PR doesn't change the API (in terms of requiring the prefix to be specified) -- I thought it'd make sense to do that in a follow-up.